src/helpers/passable_args/index.js

```diff
@@ -1,32 +1,46 @@
 // @flow
 
+/**
+ * The function which runs the validation tests.
+ *
+ * @callback passableCallback
+ * @param {function} pass
+ * @param {function} enforce
+ */
+
+/**
+ * Get Passable configuration settings
+ * specific - whitelist of tests to run
+ * passes - The function which runs the validations
+ * custom - custom validation rules
+ *
+ * @param {Array.<{specific: String[], passes: passableCallback, custom: Object}>} args - arguments for Passable configuration
+ * @return {object} Passable configuration settings
+ */
 function passableArgs(args: PassableArguments) {
 
     let passes: Function,
-        specific: specific,
-        custom: Rules | void;
-
-    if (args.length === 1) {
-        passes = args[0];
-    } else if (args.length === 3) {
-        specific = args[0];
-        passes = args[1];
-        custom = args[2];
-    } else if (args.length === 2) {
-        if (typeof args[1] === 'function') {
-            specific = args[0];
-            passes = args[1];
-        } else {
-            passes = args[0];
-            custom = args[1];
+        specific: specific = [],
+        custom: Rules = {};
+
+    switch (args.length) {
+      case 0:
+        throw new TypeError('Failed to execute `passableArgs`: At least 1 argument required, but only 0 present.')
```

Comment:
`passableArgs` would be rather cryptic to the developers looking at it, as they do not have access to the internal of the code. This line wouldn't change its meaning with `passableArgs` completely omitted. The same applies for the next case as well.

Also, don't forget a semicolon.

Reply:
This way you hint to the developer what went wrong before even going over the error stack trace.
I wouldn't omit the name of the function. That wouldn't make it better understood. 
To make it better I could add the package name.
```js
throw new TypeError('[Passable]: Failed to execute `passableArgs`: At least 1 argument required, but only 0 present.')
 ```

Reply:
Yes, it does seem to make more sense this way.